github-guidelines.md

```diff
@@ -0,0 +1,44 @@
+# GitHub Guidelines for ACT-R
+
+## New to GitHub?
+
+GitHub has excellent documentation. If you are new to GitHub, please see their documentation on how to [sign up](https://help.github.com/en/github/getting-started-with-github/signing-up-for-github), [navigate](https://help.github.com/en/github/managing-files-in-a-repository/navigating-code-on-github), [edit files](https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-another-users-repository) and [reviewing pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests). In addition to official GitHub documentation, there are a lot of excellent articles and videos on how to use GitHub.
+
+## For reviewers
+
+- Submit a review with "changes requested" if any actions are needed from the author.
+- Submit with "approved" if you agree with the change, unless someone else in your organisation already approved the pull request (1 approval per organisation). This does not prevent you from requesting changes.
+- In addition to checking that the change in the pull request is correct, check whether or not the final call period is indicated correctly in the pull request description.
+- Keep your feedback relevant to the pull request. If you have feedback unrelated to the pull request, open an issue.
```

Comment:
```suggestion
- Keep your feedback relevant to the pull request. If you have feedback unrelated to the pull request, [open an issue](https://github.com/act-rules/act-rules.github.io/issues/new).
```

This might make easier for people who are not familiar with GitHub to open an issue straight away.